bin/journey-test.sh

```diff
@@ -164,7 +164,7 @@ make_local_trackler() {
   cp -r ${track_root}/exercises tracks/${TRACK}
 
   gem install bundler
-  bundle install
+  bundle install --jobs $((`nproc --all`-1))
```

Comment:
When `nproc` is not available on a system (e.g. on macOS), produces the following log:

```
bin/journey-test.sh: line 184: nproc: command not found
+ bundle install --jobs -1
```

My guess is that this defaults back to a single job, since the build keeps running, but it might be nice to see if there are alternative ways to make this work on all the major platforms (much as we do in other parts of the test code). If that's not possible, I'd prefer to remove this. (I also wonder what happens if running on a single core; my guess is that `bundle install --jobs 0` would behave the same as `bundle install --jobs -1`, but maybe not.)